pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -32,6 +32,8 @@ const (
 	crcNetworkManagerConfigFile = "crc-nm-dnsmasq.conf"
 	// This is defined in https://github.com/code-ready/machine-driver-libvirt/blob/master/go.mod#L5
 	minSupportedLibvirtVersion = "3.4.0"
+	qemuBridgeConfigPath1      = "/etc/qemu/bridge.conf"
```

Comment:
what is the difference? One is Fedora, RHEL and the other is ... ?

Reply:
qemu is fedora, qemu-kvm is RHEL

Reply:
network or bridge?

Reply:
`s/qemuBridgeConfigPath1/qemuBridgeConfigPathFedora`
`s/qemuBridgeConfigPath2/qemuBridgeConfigPathRHEL`

Reply:
I think it was also on Debian, @cfergeau ? Hence the numbering instead of distro names.

Reply:
If across distros it would still be recommended to name them instead of numbering, and perhaps places them in a list which can be used in a `range` statement instead of code duplication? WDYT?

Reply:
I almost suggested a list yesterday, so I would not complained if one was used ;)
On Debian, it's /etc/qemu/bridge.conf, but the directory did not exist by default when I tested it. And /usr/lib/qemu/qemu-bridge-helper is not setuid.

Reply:
@gbraad I thought of an array/list but didn't go for that since there is only 2 paths. So I'll go for `_UPSTREAM` and `_RHEL` unless there is an objection.

As for dir not existing on Debian and helper not being setuid, that to me is a packaging/distro bug and IMO we should not be fixing those.

Reply:
> As for dir not existing on Debian and helper not being setuid, that to me is a packaging/distro bug and IMO we should not be fixing those.

Fairly sure the non-suid thing was discussed at length on the debian side, and if it's still done this way today (did not check), this is intentional and *not* a packaging bug.

Reply:
In that case i'd be even more reluctant for crc to touch this. Can we only check for the the setuid and not fix it but error out then if it's not setuid?

Reply:
and resort to manual steps? then what is the use of a setup command?

We inform the user WHAT we do and FIX the issue so their system MEETS PREREQUISITES.

Reply:
> I thought of an array/list but didn't go for that since there is only 2 paths

It is a better solution to use a list, and still use a named variables in the list. readability and maintainability are also key.

I hope that good example makes follow ;-)

Reply:
> We inform the user WHAT we do and FIX the issue so their system MEETS PREREQUISITES.

but @cfergeau said it's NOT an issue but a distro decision hence it's not as simple as that for me.

> It is a better solution to use a list, and still use a named variables in the list. readability and maintainability are also key.

I don't see how list is more readable with just two cases here but ok. :)

Reply:
I have not actually addressed the `setuid` issue yet. According to this blog post, setuid is not set and ` /etc/qemu/bridge.conf` need to be manually created. Do we want this addressed already in this PR or can it be a followup?

Reply:
I'll resolve this once we merge the PR and I'll create an issue for this.